pkg/build/api/v1/types.go

```diff
@@ -371,6 +375,13 @@ type SourceBuildStrategy struct {
 	ForcePull bool `json:"forcePull,omitempty" description:"forces the source build to pull the image if true"`
 }
 
+// BuildPostCommitSpec holds a build post commit hook specification.
+type BuildPostCommitSpec struct {
+	// RunBash holds a bash script to be executed in a container running the
+	// build output image.
+	RunBash string `json:"runBash,omitempty" description:"bash script to be executed in a container running the build output image"`
```

Comment:
> why isn't this a general field to run any command in the image?

Because it is meant to take `rake test` as input and run unit tests.

If it would be a generic field, the experience of the use case we're trying to address would be compromised: end users would have to pass hmm... `["bash", "-c", "rake test"]`? `["rake", "test"]` won't work in our Ruby image because we need bash to enable SCL, iow, `rake` is not in the PATH if you're not running a shell.

> also, commands should be arrays, not strings

That's a good argument for why we should not call this `RunBashCommand` as suggested above. This is not a command.